README.md

```diff
@@ -1,93 +1,104 @@
 
-Annotate the training dataset with information on: SNR, C50, pitch and speaking rate. 
+input_ids = tokenizer(description, return_tensors="pt").input_ids
+prompt_input_ids = tokenizer(prompt, return_tensors="pt").input_ids
+
+generation = model.generate(input_ids=input_ids, prompt_input_ids=prompt_input_ids)
+audio_arr = generation.cpu().numpy().squeeze()
+sf.write("parler_tts_out.wav", audio_arr, model.config.sampling_rate)
+```
 
-## Step 3: Aggregate Statistics
 
-Aggregate statistics from Step 2. Convert continuous values to discrete labels.
+## Installation steps
 
-## Step 4: Create Descriptions
+Parler-TTS has light-weight dependencies and can be installed in one line:
+```sh
+pip install parler-tts
+```
 
-Convert sequence of discrete labels to text description (using an LLM). 
+## Gradio demo
 
-## Step 5: Train the Model
+You can host your own Parler-TTS demo. First, install [`gradio`](https://www.gradio.app/) with:
+
+```sh
+pip install gradio
+```
 
-Train MusicGen-style model on the TTS task.
+Then, run:
 
+```python
+python helpers/gradio_demo/app.py
+```
+
+## Acknowledgements
+
+This library builds on top of a number of open-source giants, to whom we'd like to extend our warmest thanks for providing these tools!
+
+Special thanks to:
+- Dan Lyth and Simon King, from Stability AI and Edinburgh University respectively, for publishing such a promising and clear research paper: [Natural language guidance of high-fidelity text-to-speech with synthetic annotations](https://arxiv.org/abs/2402.01912).
+- the many libraries used, namely [🤗 datasets](https://huggingface.co/docs/datasets/v2.17.0/en/index), [🤗 accelerate](https://huggingface.co/docs/accelerate/en/index), [jiwer](https://github.com/jitsi/jiwer), [wandb](https://wandb.ai/), and [🤗 transformers](https://huggingface.co/docs/transformers/index).
+- HuggingFace 🤗 for providing compute resources and time to explore!
+
+## Contribution
+
+Contributions are welcome, as the project offers many possibilities for improvement and exploration.
+
+Namely, we're looking at ways to improve both quality and speed:
+- Datasets:
+    - Train on more data
+    - Add more features such as accents
+- Training:
+    - Add PEFT compatibility to do Lora fine-tuning.
+    - Add possibility to train without description column.
+    - Add notebook training.
+    - Explore multilingual training.
+    - Explore mono-speaker finetuning.
+    - Explore more architectures.
+- Optimization:
+    - Compilation and static cache
+    - Support to FA2 and SDPA
+- Evaluation:
+    - Add more evaluation metrics
+
+## Citation
+```
+@misc{lacombe-etal-2024-parler-tts,
+  author = {Yoach Lacombe and Vaibhav Srivastav and Sanchit Gandhi},
+  title = {Parler-TTS},
+  year = {2024},
+  publisher = {GitHub},
+  journal = {GitHub repository},
+  howpublished = {\url{https://github.com/huggingface/parler-tts}}
+}
+```
```

Comment:
Let's also add a citation for the stability paper here since we leverage their work very heavily (I can't do this as a GH suggestion due to the triple backticks that are required)

```
@misc{lyth2024natural,
      title={Natural language guidance of high-fidelity text-to-speech with synthetic annotations}, 
      author={Dan Lyth and Simon King},
      year={2024},
      eprint={2402.01912},
      archivePrefix={arXiv},
      primaryClass={cs.SD}
}
```